src/main/java/org/example/springintro/controller/OrderController.java

```diff
@@ -0,0 +1,96 @@
+package org.example.springintro.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.example.springintro.dto.order.CreateOrderRequestDto;
+import org.example.springintro.dto.order.OrderDto;
+import org.example.springintro.dto.order.OrderItemDto;
+import org.example.springintro.dto.order.OrderRequestDto;
+import org.example.springintro.dto.order.OrderStatusUpdateDto;
+import org.example.springintro.model.User;
+import org.example.springintro.services.OrderService;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.security.core.Authentication;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(
+        name = "Order",
+        description = "Endpoints for managing orders"
+)
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/api/orders")
```

Comment:
```suggestion
@RequestMapping("/orders")
```

add `server.servlet.context-path=/api` to application.properties